hwdb.d/60-keyboard.hwdb

```diff
@@ -190,7 +190,7 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnTravelMateB311RN-31*:pvr*
 
 # Travelmate C300
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnTravelMate*C3[01]0*:*
- KEYBOARD_KEY_67=f24                                    # FIXME: rotate screen
+ KEYBOARD_KEY_67=rotate_display                         # rotate screen
```

Comment:
Hmm, as mentioned in the other PR, this is now self-descriptive and thus the comments are no longer necessary?

Reply:
dunno, wanted to make the minimum changes I could get away with. i.e. just dropped the FIXME. I mean, so many other comments in that file are kinda redundant. Let's maybe clean that up another day?